lumicks/pylake/tests/test_imaging_camera/test_correlated_stack.py

```diff
@@ -201,6 +201,18 @@ def compare_frames(original_frames, new_stack):
             stack0[s]
 
 
+def test_deprecated_src():
+    fake_tiff = TiffStack(
+        [MockTiffFile(data=[np.ones((5, 4, 3))] * 6, times=make_frame_times(6))],
+        align_requested=False,
+    )
+    stack = CorrelatedStack.from_dataset(fake_tiff)
+    with pytest.warns(
+        DeprecationWarning, match="This property will be removed in a future release."
+    ):
+        stack.src
```

Comment:
Nitpicky, but it might be good to add a functional test for the forward too just to verify that it works, in case we change `_src`.
```suggestion
        stack.src
        assert id(stack.src) == id(stack._src)
```

Reply:
Good point! Done.